### PR TITLE
Changed reference of DocumentFormat.OpenXml to nuget

### PR DESCRIPTION
**Pull request by tarunchopra**

### CODE REVIEW

File: OpenXmlPowerTools.Tests/OpenXmlPowerTools.Tests.csproj [hunk 1]
```diff
@@ -33,8 +33,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="DocumentFormat.OpenXml">
-      <HintPath>..\..\Open-Xml-SDK\DocumentFormat.OpenXml\bin\Debug\DocumentFormat.OpenXml.dll</HintPath>
+    <Reference Include="DocumentFormat.OpenXml, Version=0.0.1.0, Culture=neutral, PublicKeyToken=8fb06cb64d019a17, processorArchitecture=MSIL">
```
Comment (twsouthwick) [inline OpenXmlPowerTools.Tests/OpenXmlPowerTools.Tests.csproj]: That version looks weird. I would expect that to be 2.7.2

**Comment by tarunchopra** [comment]
Did you look at taylors comment on my response ?

Thanks
TC | Sr. EE

On Feb 28, 2017, at 10:13 AM, tomjebo <notifications@github.com<mailto:notifications@github.com>> wrote:


Merged #113<https://github.com/OfficeDev/Open-Xml-PowerTools/pull/113>.

**Comment by tomjebo** [comment]
@tarunchopra are you going to change this to project.json? otherwise, I'll leave it for now (assuming you've tested).

**Comment by tarunchopra** [comment]
@twsouthwick : I'm under the impression that with introduction of vs2017, we are migrating from xproj to csproj. If this understanding is true; do we actually need this switch to xproj as you suggested?

**Pull request merged**